code/game/objects/structures/flora.dm

```diff
@@ -417,6 +417,45 @@ ICEY GRASS. IT LOOKS LIKE IT'S MADE OF ICE.
 	icon = 'icons/obj/structures/props/plants.dmi'
 	icon_state = "pottedplant_26"
 	density = FALSE
+	var/stashed_item
+	var/static/possible_starting_items = list(/obj/item/clothing/mask/cigarette/weed, /obj/item/clothing/mask/cigarette, /obj/item/clothing/mask/cigarette/bcigarette) //breaking bad reference
+	var/static/blocked_atoms = list(/obj/item/device/cotablet, /obj/item/card/id) //For things that might affect someone/everyone's round if hidden.
+	var/static/blacklist_typecache
+
+/obj/structure/flora/pottedplant/Initialize()
+	. = ..()
+
+	if(!blacklist_typecache)
+		blacklist_typecache = typecacheof(blocked_atoms)
+
+	if(prob(5))
+		var/obj/prestashed_item = pick(possible_starting_items)
+		prestashed_item = new(src)
```

Comment:
pardon my old DM rustiness but my understanding is that this will create an /obj, without the actual subtype you want.